listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -338,6 +344,53 @@ def user_feed(user_name: str):
     }})
 
 
+@user_timeline_event_api_bp.route('/user/<user_name>/feed/events/listens/following', methods=['OPTIONS', 'GET'])
+@crossdomain
+@ratelimit()
+@api_listenstore_needed
+def user_feed_listens_following(user_name: str):
+    ''' Get feed's listen events for followed users.
+
+    :param user_name: The MusicBrainz ID of the user whose timeline is being requested.
+    :type user_name: ``str``
+    :param max_ts: If you specify a ``max_ts`` timestamp, events with timestamps less than the value will be returned
+    :param min_ts: If you specify a ``min_ts`` timestamp, events with timestamps greater than the value will be returned
+    :reqheader Authorization: Token <user token>
+    :reqheader Content-Type: *application/json*
+    :statuscode 200: Successful query, you have feed listen-events!
+    :statuscode 400: Bad request, check ``response['error']`` for more details.
+    :statuscode 401: Unauthorized, you do not have permission to view this user's feed.
```

Comment:
401 is when the credentials are missing or invalid. 403 when the creds are valid but do not have access to the resource. So this needs to be fixed like:

https://github.com/metabrainz/listenbrainz-server/blob/3e4e2c216d4f000c0ef21d8fe24fc3d940b7dc38/listenbrainz/webserver/views/api.py#L656-L657

Reply:
Done